agent/engine/serviceconnect/manager_linux.go

```diff
@@ -78,9 +78,9 @@ const (
 	defaultAdminStatsRequest = httpRequestPrefix + "/stats/prometheus?usedonly&filter=metrics_extension&delta"
 	defaultAdminDrainRequest = httpRequestPrefix + "/drain_listeners?inboundonly"
 
-	defaultAgentContainerTarballPath = "/managed-agents/serviceconnect/appnet_agent.interface-v1.tar"
-	defaultAgentContainerImageName   = "appnet_agent"
-	defaultAgentContainerTag         = "service_connect.v1"
+	defaultAgentContainerImageName         = "ecs-service-connect-agent"
+	defaultAgentContainerTagFormat         = "interface-%s"
+	defaultAgentContainerTarballPathFormat = "ecs-service-connect-agent.interface-%s.tar"
```

Comment:
Sorry if my previous comment didn't make it clear - we still need the `/managed-agents/serviceconnect/` path, just that the tar file name is now `ecs-service-connect-agent.interface-%s.tar` instead of `appnet_agent.interface-%s.tar`

Reply:
my bad, I updated it incorrectly by mistake. Will fix it